servidor_graphql/index.js

```diff
@@ -0,0 +1,20 @@
+const {ApolloServer, gql } = require('apollo-server');
+
+//Definir recursos 
+const typeDefs = gql`
+    type Query {
+        hello: String
```

Comment:
qual a diferença entre colocar `String` e `String!` como retorno?
```suggestion
        hello: String!
```

Reply:
Com o string se estiver vazio ele retornará null, mas com o string! ele retornará o vazio [].

Reply:
O String permite que tenhamos como retorno o valor null, mas o String! obriga que tenhamos como retono uma string e se não tiver será vazio